fortls/parsers/internal/intrinsics.py

```diff
@@ -277,7 +277,9 @@ def update_m_intrinsics():
             val = val.replace(f"**{key.upper()}**(3)", f"**{key.upper()}**")
             markdown_intrinsics[key] = val
 
-        with open("fortls/intrinsic.procedures.markdown.json", "w") as f:
+        with open(
+            "fortls/parsers/internal/intrinsic.procedures.markdown.json", "w"
+        ) as f:
```

Comment:
This should probably use `pathlib` now.

```py
from pathlib import Path
...

with open(Path(__file__).parent / "intrinsic.procedures.markdown.json", "w" as f):
    ...
```

Reply:
Yeah that will improve flexibility and readability